kong/plugins/log-serializers/influxdb.lua

```diff
@@ -0,0 +1,30 @@
+local _M = {}
```

Comment:
If you reuse the http-log code, the size of the `handler.lua` file will shrink considerably, so maybe you could even incorporate the `serialize` function into that file.